### ISSUE TITLE
XLS Spreadsheet file are not linked

### ISSUE THREAD

**Issue by laggingreflex** [alxnbl/onenote-md-exporter#40]
**Describe the bug**

A page with an XLS Spreadsheet file isn't linked up correctly.

The XLS file in the resources folder is created without the .xls extension

The link in the note is only to the preview image (PNG), not the actual spreadsheet (XLS) file. 

The exported note appears as:

```md
Page - Spreadsheet.xlsx

![282f526f39dc4a60b258f1be0363491c](:/282f526f39dc4a60b258f1be0363491c)
```
The second line is preview image. 

**To Reproduce**

Create a note with an XLS spreadsheet. Then export. 

**Expected behavior**

The note should have a link to the XLS spreadsheet. 

Or it should probably have link to both - the preview as well as the spreadsheet.

**Comment by alxnbl** [comment]
Hey @laggingreflex and thank you for describing this bug. Could you try to reproduce with version 1.0 please ?

**Comment by alxnbl** [comment]
I close the issue which is very probably fixed in 1.0 release.
